mne/preprocessing/artifact_detection.py

```diff
@@ -13,21 +13,19 @@
 
 
 @verbose
-def annotate_muscle_zscore(raw, threshold=4, picks=None, min_length_good=0.1,
+def annotate_muscle_zscore(raw, threshold=4, ch_type=None, min_length_good=0.1,
                            filter_freq=(110, 140), n_jobs=1, verbose=None):
-    """Detect segments that likely contain muscle artifacts.
+    """Create annotations for segments that likely contain muscle artifacts.
 
     Detects data segments containing activity in the frequency range given by
-    ``filter_freq``, that exceeds the specified z-score threshold.
-
-    Raw data is band-pass filtered between ``filter_freq`` especified
-    frequencies (default is 110 - 140 Hz), the signal envelope computed,
-    z-scored across samples, channel summation and division by the square root
-    of the channel number, and low-pass filtered to better capture beginning
-    and end of muscle activity and false positive transient peaks.
-
-    .. note::
-        Use a single channel type.
+    `filter_freq` whose envelope magnitude exceeds the specified z-score
+    threshold (when summed across channels and divided by `sqrt(n_channels)`).
+    False-positive transient peaks are prevented by low-pass filtering the
+    resulting z-score time series at 4 Hz. Only operates on a single channel
+    type, if `ch_type` is ``None`` it will select the first type in the list
+    ``mag`` ``grad`` ``eeg``.
```

Comment:
since you'll need to edit this paragraph anyway: please add commas between these items